Source/WebCore/inspector/InspectorThreadableLoaderClient.h

```diff
@@ -39,6 +39,10 @@ class ThreadableLoader;
 
 namespace Inspector {
 
+#if ENABLE(MALLOC_HEAP_BREAKDOWN)
+using WebCore::LoaderMalloc;
+#endif
+
```

Comment:
Umm, this doesn't look nice to me. No other code does this tricky hack. I think the second patch looks better. WDYT?
https://github.com/WebKit/WebKit/commit/310a7fbf859a335d18851f31a18e38ed69b16287